### PR TITLE
system_info message added to mmbwmon.

### PR DESCRIPTION
**Pull request by jbreitbart**
This message contains all measurements of that system and its config.

### CODE REVIEW

File: src/message/agent/mmbwmon/system_info.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+namespace mmbwmon {
+
+system_info::system_info(const size_t _threads, const size_t _smt, const size_t _numa, const std::vector<double> &_membw): threads(_threads), smt(_smt), numa(_numa), membw(_membw)
+{
+}
+
+YAML::Node system_info::emit() const
+{
+	YAML::Node node;
+	node["task"] = "mmbwmon system info";
```
Comment (spickartz) [inline src/message/agent/mmbwmon/system_info.cpp]: Isn't that redundant information? You already have the task encoded in the topic.

File: src/message/agent/mmbwmon/system_info.cpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	return node;
+}
+
+void system_info::load(const YAML::Node &node)
+{
+	fast::load(threads, node["threads"]);
+	fast::load(smt, node["smt"]);
+	fast::load(numa, node["numa"]);
+	fast::load(membw, node["bandwidth"]);
+}
```
Comment (spickartz) [inline src/message/agent/mmbwmon/system_info.cpp:28]: If it isn't redundant you should load it here.

Comment (jbreitbart) [reply src/message/agent/mmbwmon/system_info.cpp:28]: It is not stored in the struct as it is always the same so it can't be loaded. ;)

**Comment by jbreitbart** [comment]
Should all be fixed.

**Pull request merged**